fiftyone/core/video.py

```diff
@@ -142,6 +142,22 @@ def name(self):
     def media_type(self):
         return fom.IMAGE
 
+    def _get_sample_only_fields(
+        self, include_private=False, use_db_fields=False
+    ):
+        sample_only_fields = set(
+            self._get_default_sample_fields(
+                include_private=include_private, use_db_fields=use_db_fields
+            )
+        )
+
+        # If sample_frames != dynamic, `filepath` can be synced
+        config = self._frames_stage.config or {}
+        if config.get("sample_frames", None) != "dynamic":
```

Comment:
nit: alternatively you could return early if and remove this if
```
...
if config.get("sample_frames", None) == "dynamic":
  return
... rest ...
```

Reply:
Hmm yeah could go either way